code/web/src/modules/user/EditProfile.js

```diff
@@ -32,14 +34,21 @@ class EditProfile extends Component {
     }
 
     showEditInfo = () => {
-        //this is the function that will show and hide the menu
+        if (this.state.shown === true) {
+            this.setState({ shown: false })
+        } else {
+            this.setState({ shown: true }) 
+        }
+       
```

Comment:
The edit menu is shown based on the state of h component.

Reply:
Nice, this is looking good!